### ISSUE TITLE
bcachefs show-super errors

### ISSUE THREAD

**Issue by jpsollie** [koverstreet/bcachefs-tools#29]
I saw an issue when I performed a "show-super" on the HDD device (bcachefs show-super /dev/sdb1):
 
`External UUID:                  9cac6a05-1bd9-422a-9b93-2a2c53dc6e51
Internal UUID:                  0781f15e-09b0-4ef2-967a-fb499dd0f9ac
Label:
Version:                        11
Created:                        Wed May 20 08:01:51 2020
Block_size:                     512
Btree node size:                256.0K
Error action:                   remount-ro
Clean:                          0
Features:                       atomic_nlink,new_siphash,new_extent_overwrite,btree_ptr_v2,extents_above_btree_updates,btree_updates_journalled
Metadata replicas:              1
Data replicas:                  1
Metadata checksum type:         crc32c (1)
Data checksum type:             crc32c (1)
Compression type:               none (0)
Foreground write target:        Group 0 (ssd)
Background write target:        Group 1 (hdd)
Promote target:                 Group 0 (ssd)
String hash type:               siphash (2)
32 bit inodes:                  0
GC reserve percentage:          8%
Root reserve percentage:        0%
Devices:                        2 live, 2 total
Sections:                       journal,members,replicas_v0,disk_groups,clean
Superblock size:                11128

Members (size 120):
  Device 0:
    UUID:                       b544ac1f-de93-4555-bd71-4724bd6f641c
    Size:                       107.3G
    Bucket size:                256.0K
    First bucket:               0
    Buckets:                    439556
    Last mount:                 Wed May 20 08:14:01 2020
    State:                      readwrite
    Group:                      ssd (0)
    Data allowed:               journal,btree,data
    Has data:                   (none)
    Replacement policy:         lru
    Discard:                    1
  Device 1:
    UUID:                       3970eff4-f633-43f0-9933-fdf0f39e092f
    Size:                       3.6T
    Bucket size:                512.0K
    First bucket:               0
    Buckets:                    7618540
    Last mount:                 Wed May 20 08:14:01 2020
    State:                      readwrite
    Group:                      hdd (1)
    Data allowed:               journal,btree,data
    Has data:                   (none)
    Replacement policy:         lru
    Discard:                    1
`
To summarize: Discard is not active for HDD.  I am 100% sure I did not enter that when performing the format operation0. Did format make the error? or is it an error in show-super?

**Comment by koverstreet** [comment]
Closing as stale — opened 2020-05-20 with no follow-up. bcachefs-tools has been substantially rewritten since (Rust conversion, new show-super path). If you still see this on a current build, please reopen with fresh output.
